hlp2b.mml

```diff
@@ -1,63 +1,22 @@
-<Font|promptFont>ALLEGIANCE ZONE GAMES<Font|smallFont><p><p>
+<Font|promptFont>MOTD and community events<Font|smallFont><p><p>
 
-The Allegiance Zone is your gateway to the subscription part of
-Allegiance. The Allegiance Zone offers persistent rankings for your own
-player character, for any registered squad that you may create or be a
-member of, and for any other player or squad. The Allegiance Zone
-also offers unique game events, custom factions that only Allegiance Zone
-subscribers can play, and the largest multiplayer games of all three
-connection options.<p><p>
 
-<Font|promptFont>Message of the Day<Font|smallFont><p><p>
-
-Message of the Day provides the latest information on
+The Message of the Day provides the latest information on
 Allegiance. The message will be routinely updated with the latest games, events,
-and offerings brought to you by the MSN Gaming Zone.<p><p>
+and offerings brought to you by the Allegiance community.<p><p>
 
 <Font|promptFont>Games List<Font|smallFont><p><p>
 
 Click <Font|smallBoldFont>Games List<Font|smallFont> to display all games currently running
-on the Allegiance Zone servers. For detail on how to connect to and join an Allegiance Zone
+on the public servers. For detail on how to connect to and join a public game 
 game, access Connect & Start: <Main|hlp2g|Joining a Game.><p><p>
 
-<Font|promptFont>Zone Events<Font|smallFont><p><p>
-
-Click <Font|smallBoldFont>Zone Events<Font|smallFont> to display a
-schedule of all upcoming events hosted by the MSN Gaming Zone.<p><p>
-
-<Font|promptFont>Squad Profiles<Font|smallFont><p><p>
-
-Click <Font|smallBoldFont>Squad Profiles<Font|smallFont> to access all squad statistics and
-squad membership lists as well as to create and manage your own squads. For detailed
-information, access Allegiance Zone: <Main|hlp3b|Squad Profiles.><p><p>
-
-<Font|promptFont>Leader Board<Font|smallFont><p><p>
+<Font|promptFont>Community Events <Font|smallFont><p><p>
 
-Click <Font|smallBoldFont>Leader Board<Font|smallFont> to display the rankings and statistics
-of all Allegiance Zone members and to access the Player Profiles of all other Allegiance Zone
-members. For detailed information, access Allegiance Zone: <Main|hlp3a|Leader Board.><p><p>
+Check the steam community hub and our dscord sever  to display a
```

Comment:
- Uppercase Steam
- "dscord" -> "Discord"
- "sever" -> "server"
- double space after server

Reply:
Should we also include a link to the Discord server?